sbt-plugin/src/sbt-test/incremental/change-config/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.3.10
```

Comment:
Is this deliberately different from other versions in `build.properties`?

Reply:
No, there's actually no reason for this file, it's an artifact from having tried to run sbt manually on the command line (which doesn't work). Fixed in 05d8a6791c30eeeb4c4a3f4a01447e71b6ac387a.